src/Query/Builder.php

```diff
@@ -659,6 +660,26 @@ public function whereBetween($column, iterable $values, $boolean = 'and', $not =
 
     /** @inheritdoc */
     public function insert(array $values)
+    {
+        return $this->performInsert($values);
+    }
+
+    /** @inheritdoc */
+    public function insertOrIgnore(array $values)
```

Comment:
I'll open a distinct PR for this method that is no longer required. It was used in [`DatabaseStore::add`](https://github.com/laravel/framework/blob/ed718c7c7de17cad775e0eabbd441238fa7d32ec/src/Illuminate/Cache/DatabaseStore.php#L164), but I replaced the implementation for MongoDB.

Reply:
See PHPORM-170